src/Components/Web.JS/src/DomWrapper.ts

```diff
@@ -5,9 +5,15 @@ export const domFunctions = {
   focusBySelector,
 };
 
-function focus(element: HTMLElement, preventScroll: boolean): void {
+function focus(element: HTMLOrSVGElement, preventScroll: boolean): void {
   if (element instanceof HTMLElement) {
     element.focus({ preventScroll });
+  } else if (element instanceof SVGElement) {
+    if (element.hasAttribute('tabindex')) {
+      element.focus({ preventScroll });
+    } else {
+      throw new Error('Unable to focus an invalid SVG element because it does not have a tabindex.');
```

Comment:
Nit:
```suggestion
      throw new Error('Unable to focus an SVG element that does not have a tabindex.');
```
(The SVG element itself is not invalid, but it was just invalid to attempt to focus it.)